README.md

```diff
@@ -193,7 +195,71 @@ const outputs = [
 ];
 ```
 
-Note that syft.js doesn't handle user's data collection, data storage and loading.
+### Stop & Resume
+`PlanTrainer` allows stopping and resuming the training using `stop` and `resume` methods:
+
+```javascript
+  // Main training loop.
+  const training = job.train('training_plan', {
+    inputs: [/* ... */],
+    outputs: [/* ... */],
+    data,
+    target,
+  });
+
+  training.on('start', () => {
+    // training is started!
+  });
+
+  training.on('stop', () => {
+    // training is stopped!
+  });
+
+  document.getElementById('stop-button').onclick = () => {
+    training.stop();
+  };
+
+  document.getElementById('resume-button').onclick = () => {
+    training.resume();
+  };
+```
+
+### Checkpointing
+
+`stop` method returns current training state as `PlanTrainerCheckpoint` object, 
+which can be serialized to JSON to restored from JSON later to continue the training:
+
+```javascript
+const checkpoint = await training.stop();
+const checkpointJson = await checkpoint.toJSON();
+localStorage.setItem('checkpoint', JSON.stringify(checkpointJson));
+
+// ... checkpoint can survive page reload ...
+
+const checkpointJsonString = localStorage.getItem('checkpoint');
+const checkpointJson = JSON.parse(checkpointJsonString);
```

Comment:
You should make this step internal to the fromJSON method. No need to get the user to do this.